### ISSUE TITLE
Java not installed when existing Java is incompatible

### ISSUE THREAD

**Issue by jrpool** [validator/validator#1997]
### Summary

The postinstaller does not run when the existing Java is incompatibly old. Then running `vnu.check` throws an `Exception in thread "main" java.lang.UnsupportedClassVersionError: nu/validator/client/SimpleCommandLineValidator : Unsupported major.minor version`. The documentation does not warn users that they need to update Java.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Spec links/citations

n/a

### Steps to reproduce

1. Install `vnu-jar` on a host that already has an incompatibly old version of Java.
2. Execute `vnu.check`.

### Expected behavior

Postinstaller runs to install a compatible Java in `node_modules/.cache`.

### Actual behavior

Crash with error thrown.

### Screenshots or screen recordings

n/a

**Comment by sideshowbarker** [comment]
Jonathan, thanks for catching this and reporting it. I’ve made a fix and published a v25.12.22 of the package.

Please give it a try — and if it still doesn’t work as expected, lemme know, and we can re-open this.

**Comment by jrpool** [comment]
Thank you for the swift improvement, @sideshowbarker. I have tested it and verified that the postinstaller now operates when an incompatibly old version of Java is installed.
